src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -785,4 +574,217 @@ public static Image getGameIcon(final Window frame) {
     }
     return img;
   }
+
+  public static void startNewTripleA(final Long maxMemory) {
+    startGame(System.getProperty(GameRunner.TRIPLEA_GAME_PROPERTY), null, maxMemory);
+  }
+
+  public static void startGame(final String savegamePath, final String classpath, final Long maxMemory) {
+    final List<String> commands = new ArrayList<>();
+    if (maxMemory != null && maxMemory > (32 * 1024 * 1024)) {
+      ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath, maxMemory);
+    } else {
+      ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath);
+    }
+    if (savegamePath != null && savegamePath.length() > 0) {
+      commands.add("-D" + GameRunner.TRIPLEA_GAME_PROPERTY + "=" + savegamePath);
```

Comment:
Yes, but not going to change right now as part of this PR. The code was imported wholesale from one class to another to allow for easier diffing. Some obvious next steps are to fix up this code, there is much to fix. It's not within the scope of this PR. 

Given the move of code, a good question would be if GameRunner is a better home for this code at this point in time? IMO yes as demonstrated by the variable dependency graph, merging the two allowed variables to go from public or protected to private.